web/pages/api/v0/search-markets.ts

```diff
@@ -12,6 +12,18 @@ export default async function handler(
   const { terms } = req.query
   const keywords = terms as string
 
-  const contracts = await searchContracts(keywords, 100)
-  res.status(200).json(contracts.map((c) => toFullMarket(c)))
+  const { fuzzyOffset: _fuzzyOffset, data } = await searchContract({
+    state: {
+      contracts: undefined,
+      fuzzyContractOffset: 0,
+      shouldLoadMore: false,
+      showTime: null,
+    },
```

Comment:
Should create this default state within `supabase/contract.ts`. You could even make it the default argument value of state, e.g. function searchContract(query: string, filter: string, ... state = defaultState) { ... }